dao/tenant_dao.go

```diff
@@ -19,3 +19,13 @@ func GetOrCreateTenantID(accountNumber string) (*int64, error) {
 
 	return &tenant.Id, result.Error
 }
+
+func TenantBy(accountNumber string) (*m.Tenant, error) {
```

Comment:
Rather than using the default parameter name as an extension of the function (not all of us use jetbrains :stuck_out_tongue_closed_eyes: ) can we rename this `TenantByAccountNumber`

Reply:
:) I renamed and I will keep suggested pattern next time